tests/fixtures/tests.py

```diff
@@ -1233,6 +1322,32 @@ def test_format_discovery(self):
             ],
         )
 
+    def test_force_insert(self):
+        """use force_insert option to reduce query count"""
+        from django.db import connection
+
+        with self.settings(DEBUG=True):
```

Comment:
Use `CaptureQueriesContext` at the very least.

Reply:
This us realky useful, but i could not find CaptureQueriesContext in django documentation. Shall i add this?